{{cookiecutter.repo_name}}/README.rst

```diff
@@ -9,53 +13,50 @@
 Setup
 =====
 
-Using pip
----------
-
-The preferred way to install this repo is via SSH.  This command will install the latest version of `{{ cookiecutter.project_name }}`, provided you have `SSH set up on GitHub`_.
-
-.. code-block:: console
+For detailed installation instructions, see :ref:`installation`.
 
-    $ pip install git+ssh://git@github.com/anmut-consulting/{{ cookiecutter.repo_name }}.git@latest
+Source Code
+===========
 
-.. seealso::
+The {{ cookiecutter.project_name }} source code can be found in the `Github repo`_ or in the `Distributions`_ folder within the Anmut Hub - Software Products SharePoint.
 
-	For more detailed installation instructions, see :ref:`installation`
+For more on installing from source, see :ref:`installation`.
 
-.. _SSH set up on GitHub: https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh
```

Comment:
I think it's important to leave this part in as it seems many people are not aware of how to do the SSH setup.  They don't even know about the option...